core/src/main/java/hudson/PluginManager.java

```diff
@@ -211,11 +211,18 @@ public abstract class PluginManager extends AbstractModelObject implements OnMas
      */
     /* private final */ static int CHECK_UPDATE_ATTEMPTS;
 
+    /**
+     * List of detached plugins that should not be installed as implied dependency.
+     */
+    @SuppressFBWarnings(value = "MS_MUTABLE_COLLECTION_PKGPROTECT", justification = "for script console")
```

Comment:
Can be private if that's the problem, the script console doesn't care about visibility.